requirements.txt

```diff
@@ -20,5 +20,6 @@ raven[flask] == 5.9.2
 setproctitle == 1.1.9
 six == 1.9.0
 SQLAlchemy == 1.0.9
+urllib3[secure]==1.19
```

Comment:
my understanding is that [secure] pulls these in

http://urllib3.readthedocs.io/en/latest/user-guide.html#certificate-verification-in-python-2 "if you want to install the packages manually..."